applications/matter_bridge/README.rst

```diff
@@ -493,6 +499,16 @@ After building the sample and programming it to your development kit, complete t
 
                ./chip-tool onoff read on-off *<bridge_node_ID>* *<bridged_device_endpoint_ID>*
 
+
+            Read the value of the *current-position* attribute from the *switch* cluster using the following command:
+
+            .. parsed-literal::
+               :class: highlight
+
+               ./chip-tool switch read current-position *<bridge_node_ID>* *<bridged_device_endpoint_ID>*
+
+            Note that the Generic Switch implements the momentary switch which, in contrast to the latching switch, remains switched on only as long as the physical button is compressed.
```

Comment:
```suggestion
            Note that the Generic Switch implements the momentary switch.
            This means that, in contrast to the latching switch, it remains switched on only as long as the physical button is pressed.
```
Is it correct to say "implements the momentary switch", or should it be "is implemented as a momentary switch"?

Reply:
I'm not sure, for me both versions seem correct. I can use the latter, no problem.